operations/loyaltymemberships.md

```diff
@@ -0,0 +1,280 @@
+            "Url": "https://www.mews.com/"
+        },
+        {
+            "Id": "ea7da00f-fdc9-4014-b0f7-71003b87e3d0",
+            "LoyaltyProgramId": "8a98965a-7c03-48a1-a28c-ab1b009b53c8",
+            "AccountId": "0ed43ab7-4592-4c99-906a-426588de1c00",
+            "Code": "Code-002",
+            "ExpirationDate": "",
+            "Url": ""
+        }
+    ],
+    "Cursor": "ea7da00f-fdc9-4014-b0f7-71003b87e3d0"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `LoyaltyMemberships` | array of [Loyalty memberships](#loyalty-membership) | required | The loyalty memberships of the enterprise. |
+| `Cursor` | string | optional | Unique identifier of the item one newer in time order than the items to be returned. If Cursor is not specified, i.e. null, then the latest or most recent items will be returned. |
+
+#### Loyalty membership
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the loyalty membership. |
+| `LoyaltyProgramId` | string | required | Unique identifier of the loyalty program. |
+| `AccountId` | string | required | Unique identifier of the account. |
+| `Code` | string | required | Code of the loyalty membership. |
+| `ExpirationDate` | string | optional | Expiration date of the loyalty membership in UTC timezone in ISO 8601 format. |
+| `Url` | string | optional | Url of the loyalty membership. |
+
+## Add loyalty memberships
+
+Adds loyalty memberships to the enterprise.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/loyaltyMemberships/add`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "LoyaltyMemberships": [
+        {
+            "LoyaltyProgramId": "3ed9e2f3-4bba-4df6-8d41-ab1b009b6425",
+            "AccountId": "87d4c7c4-4832-4341-8b54-e45c1a73df34",
+            "Code": "Code-003",
+            "ExpirationDate": "2022-12-31",
+            "Url": ""
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `LoyaltyMemberships` | array of [Loyalty membership parameters](#loyalty-membership-parameters) | required, max 1000 items | Loyalty memberships to be added. |
+
+### Loyalty membership parameters
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `LoyaltyProgramId` | string | required | Unique identifier of the loyalty program whose member the enterprise is. |
+| `AccountId` | string | required | Unique identifier of the account whose member the enterprise is. |
```

Comment:
This wording seems weird, I'd understand it like the enterprise is member of a program 😕

Reply:
Yeah it's not very clear 🤔 What about; `Unique identifier of the loyalty program that belongs to the enterprise.`?
We can skip mentioning the enterprise if it's not that crucial?

Reply:
I'd omit it all together since Connector API is enterprise scoped and therefore it's implied.